k8sfv/pod.go

```diff
@@ -226,3 +241,31 @@ func cleanupAllPods(clientset *kubernetes.Clientset, nsPrefix string) {
 var zeroGracePeriod int64 = 0
 
 var deleteImmediately = &v1.DeleteOptions{GracePeriodSeconds: &zeroGracePeriod}
+
+func runCommand(command string, args ...string) error {
```

Comment:
Could you use `Cmd.CombinedOutput()`, or are you keen to keep stdout/err separate?